db/migrate/20211118070934_add_new_columns_to_location.rb

```diff
@@ -0,0 +1,7 @@
+class AddNewColumnsToLocation < ActiveRecord::Migration[6.1]
+  def change
+    add_column :locations, :start_date, :datetime
+    add_column :locations, :end_date, :datetime
+    add_column :locations, :exclude_dates, :string
```

Comment:
Perhaps this would be better:

```
    add_column :locations, :start_date, :date
    add_column :locations, :end_date, :date
    add_column :locations, :exclude_dates, :text, array: true, default: []
```